health_monitor/health_monitor.ino

```diff
@@ -0,0 +1,167 @@
+#include <BLEDevice.h>
+#include <string>
```

Comment:
Double check that `string_view` isn't available... it might actually be. I was surprised at how many c++ features were actually available last time I used the arduino stuff.

Reply:
```
/home/griswald/ws/src/superspreader/health_monitor/health_monitor.ino:23:22: note: 'std::string_view' is only available from C++17 onwards
```
![](https://media.giphy.com/media/wYyTHMm50f4Dm/giphy-downsized.gif)